msdk-io/msdk-io-mzml/src/test/java/io/github/msdk/io/mzml/MzMLFileExportMethodTest.java

```diff
@@ -246,7 +246,7 @@ public void testFileWithUV() throws MSDKException, IOException {
         scan.getNumberOfDataPoints());
     Assert.assertEquals(Integer.valueOf(2114), scan.getScanNumber());
     Assert.assertEquals(MsSpectrumType.CENTROIDED, scan.getSpectrumType());
-    Assert.assertEquals(9.939699e06f, scan.getTIC(), 0.0001);
+    Assert.assertEquals(9.939699e06f, scan.getTIC(), 1e01);
```

Comment:
The expected value is what is found in the mzML file. The computed value surprisingly has an error of several magnitudes higher.
This seems to be a recurring issue with the parsed intensity values, and might have to do something with the conversion from 64-bit to 32-bit precision. I will have a look into the float decoder to make sure there isn't a problem with it.